scripts/ci/run-jenkins.sh

```diff
@@ -94,6 +94,15 @@ then
 	wget -qO- https://download.mono-project.com/test/new-certs.tgz| tar zx -C ~/.config/.mono/
 fi
 
+
+if [[ ${CI_TAGS} == *'linux-'* ]];
```

Comment:
I'd prefer if we made this an explicit `CI_TAGS == *'ccache'*` so we don't turn it on everywhere (i.e. it will probably cause issues on jobs that already override the compiler like the static analysis ones).

Reply:
Another thing we need to solve is where to put the ccache cache, cause as far as I know it gets put into $HOME by default which would be cleaned up after the build when the chroot is destroyed.

@directhex can help us figure out a location that is preserved.

Reply:
We can add a bind-mounted directory to the job config. Android uses this to share android-toolchain between different jobs.